File: python/01_08_pares_impares/pares_impares.py
'''
Programa : 01_08_pares_impares
El programa tomas 2 valores enteros, si la es divisible entre 2 o si el residuo de la división
es 0, interpreta que el numero es par caso contrario impar y lo muestra por pantalla.

__author__ = "Alvaro juanez M"
__version__ = "1.0"
__status__ = "Aprendiendo python"
'''
import sys #Modulo libreria, da acceso a funciones especificas del sistema


numero = int(sys.argv[1]) #metodo para introducir un valor como argumento por consola

#Verifica si el numero es par o impar
if numero % 2 == 0:
    print("El numero ", numero, " es par")
else:
    print("El numero ", numero, " es impar")

File: java/01_07_Pares_Impares/ParesImpares.java
/*
 * ParesImpares: El programa pide al usuario que ingrese un numero como argumento si es par le dice que es par,
 * de lo contrario le dira que es impar
 * @author: Alvaro Juanez M
 * @version: 1.0 26/01/2023
 *
 */
import java.util.Scanner;

public class ParesImpares{
    public static void main(String[] args){

        Scanner scanner = new Scanner(args[0]);   //Entrada de datos
        
        //Guarda el valor de las variables
        int numero;

        //Inicia el Programa, interactua con el usuario
        numero = scanner.nextInt(); 

        scanner.close(); //Cierre scanner

        //Si es par
        if(numero % 2 == 0){
            System.out.println("El numero es Par");
        }else{
            System.out.println("El numero es Impar");
        }
    }
}

